classifier_and_summarizer/summarizer/budgeted_max_cover/cover.py

```diff
@@ -0,0 +1,18 @@
+''' Module to define the Cover object used
+in budgeted max cover solver
+'''
+
+
+class Cover:
+    '''
+    Boiled down representation of a stamp page consists
+    of only the cover of elements along with the cost and id.
+
+    All elements are considered to have equal weight.
+    '''
```

Comment:
Add a desciption for class variables as well?

Reply:
Updated accordingly.